ui/readme-terra-ui-packages.md

```diff
@@ -0,0 +1,29 @@
+# Using Packages from @terra-ui-packages
+
+Terra UI publishes a [set of packages](https://github.com/DataBiosphere/terra-ui/tree/dev/packages) on every commit. These are stored in a public registry for use by anyone. We can and do use packages from this registry in our codebase.
+
+## Using Packages
+
+The [packages](https://github.com/DataBiosphere/terra-ui/tree/dev/packages) directory in the [Terra UI GitHub repo](https://github.com/DataBiosphere/terra-ui) contains a set of directories each with its own package.json file. Here is a current (2023-08-31) listing:
+```
+$ grep -R \"name\": packages
+packages/test-utils/package.json:  "name": "@terra-ui-packages/test-utils",
+packages/components/package.json:  "name": "@terra-ui-packages/components",
+packages/core-utils/package.json:  "name": "@terra-ui-packages/core-utils",
+packages/build-utils/package.json:  "name": "@terra-ui-packages/build-utils",
+```
+
+These packages are published at a Broad-owned registry. Because this registry is public, the JSON package descriptions are available:
+```
+$ curl https://us-central1-npm.pkg.dev/dsp-artifact-registry/terra-ui-packages/@terra-ui-packages/core-utils
+{"name":"@terra-ui-packages/core-utils","dist-tags":{"latest":"0.0.5"},...}
+```
+
+The `gitHead` field contains the commit corresponding to the published version. I find it useful to browse the Terra UI source tree at the version's commit to see exactly which functions/components are available in the published version. For example, @terra-ui-packages/core-utils at version 0.0.5 has a gitHead (git hash) c79428c00ec8c32994294257ffb628e3d1d9b10f. The source tree at that commit is available here:
```

Comment:
This clause was a little confusing to me:
`has a gitHead (git hash)`

Is this saying that core-utils has a release (0.0.5) and that release has a hash of: c79428c00ec8c32994294257ffb628e3d1d9b10f

I think that it is mainly the term gitHead that is confusing me.

Reply:
Would "git commit hash" be clearer?

Reply:
I wonder if hash of a git commit would be better.

Any time I see git .... I think it is a command, and then I get confused that document is asking me to run a git command.

Reply:
Nice!